src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -510,6 +524,40 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
                     LeaderShipDuration = leader?.LeaderShipDuration,
                 });
             }
+
+            void DismissUnrecoverableNotification()
+            {
+                try
+                {
+                    serverStore.NotificationCenter.Dismiss(AlertRaised.GetKey(AlertType.UnrecoverableClusterError, null), context.Transaction, false);
```

Comment:
Let's make the key here more unique (using `term/raft-index`), so I'll will add/dismiss the proper `UnrecoverableClusterError` alert

Reply:
done